cedar-drt/fuzz/src/lib.rs

```diff
@@ -57,6 +60,22 @@ impl<'e> DifferentialTester<'e> {
         }
     }
 
+    /// Differentially test evaluating the given expression.
+    /// `r` and `entities` are used to populate the evaluator.
+    pub fn run_eval_test(&self, r: &ast::Request, expr: &Expr, entities: &Entities) -> bool {
+        let exts = Extensions::all_available();
+        let eval = match Evaluator::new(r, entities, &exts) {
```

Comment:
Probably want the use of extensions restricted by a flag (like we have for `ABACSettings`)